src/ui/Calendar/PresetCalendar.svelte

```diff
@@ -35,10 +39,10 @@
     },
   ]
 
-  export let date = [new Date(), new Date()]
+  export let date: DateRange = [new Date(), new Date()]
```

Comment:
Initial `date` property is required to have tuple of `2` dates, due to `range` calendar mode. If tuple with a single date is passed, calendar will await selection of the second date, which might be considered not good UX.
Solution: strict type `[Date, Date]`

Reply:
Fixed